documentation/modules/exploit/windows/iis/iis_webdav_upload_asp.md

```diff
@@ -0,0 +1,45 @@
+## Description
+
+This module can be used to execute a payload on IIS servers that have world-writeable directories. The payload is uploaded as an ASP script via a WebDAV PUT request. A WebDAV PUT request is used for uploading file on webdav server. 
```

Comment:
The second sentence is repetitive, I'd remove it, i think the first sentence is solid.